tests/Services/SearchServiceTest.php

```diff
@@ -194,8 +197,10 @@ public function testItemVariantIdExtractingForRedirectUrlGeneration()
 
         $itemSearchServiceMock = $this->getMockBuilder(ItemSearchService::class)
             ->disableOriginalConstructor()
-            ->setMethods([])
             ->getMock();
+        $itemSearchServiceMock->expects($this->once())
+            ->method('getResult')
+            ->willReturn(['success' => false, 'total' => 0]);
```

Comment:
Returning 'success' shouldn't be necessary anymore

Reply:
Same as above